### ISSUE TITLE
Seems like a great idea.

### ISSUE THREAD

**Issue by RyPoints** [marlonjames71/DropDownMenu#1]
Seems like a great idea.  I was about to look at it on MacOS, but the DropDownMenu isn't in the code.

**Comment by marlonjames71** [comment]
You're correct. It isn't on here yet as I am trying to turn it into a tutorial video for YouTube. Currently, the video is too long. Once I get it shorter, I will add the code needed to get going as well as the code for the final project.

@RyPoints Thanks for the inquiry.
